bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo_it.properties

```diff
@@ -459,7 +459,7 @@ config.refreshInterval.description = Intervallo di aggiornamento per interrogare
 
 conf-error-no-client-id = Impossibile connettersi al bridge Netatmo poiché non è configurato il Id cliente
 conf-error-no-client-secret = Impossibile connettersi al bridge Netatmo poiché non è configurata alcuna parola segreta
-conf-error-grant-needed = Completa la configurazione autorizzando il binding a Netatmo Connect\: ''{0}''.
+conf-error-grant-needed = Completa la configurazione autorizzando il binding a Netatmo Connect\: {0}
```

Comment:
Please revert this change, it will have to be done at Crowdin, see https://v40.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
I have removed this from the commit